pkgs/development/python-modules/nlpo3/default.nix

```
@@ -0,0 +1,64 @@
+{
+  lib,
+  buildPythonPackage,
+  rustPlatform,
+  fetchFromGitHub,
+  pythonOlder,
+  pythonAtLeast,
+
+  rustc,
+  cargo,
+
+  setuptools,
+  setuptools-rust,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "nlpo3";
+  version = "1.3.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6" || pythonAtLeast "3.12";
```

```suggestion
  disabled = pythonAtLeast "3.12";
```

We don't have python older than 3.6 in-tree